File: python/abc173/b.py
n = int(input())

ac = 0
wa = 0
tle = 0
re = 0

for _ in range(n):
    i = input()

    if i == "AC":
        ac += 1
    if i == "WA":
        wa += 1
    if i == "TLE":
        tle += 1
    if i == "RE":
        re += 1

print("AC x {}".format(ac))
print("WA x {}".format(wa))
print("TLE x {}".format(tle))
print("RE x {}".format(re))

File: rust/abc173/src/bin/b.rs
use proconio::{fastout, input};
use std::collections::HashMap;

#[fastout]
fn main() {
    input! {
        n: isize,
        list: [String; n],
    }

    let mut result = HashMap::new();
    for s in list {
        let s = s.to_string();
        match &*s {
            "AC" => {
                (*result.entry("AC").or_insert(0)) += 1;
            }
            "WA" => {
                (*result.entry("WA").or_insert(0)) += 1;
            }
            "TLE" => {
                (*result.entry("TLE").or_insert(0)) += 1;
            }
            "RE" => {
                (*result.entry("RE").or_insert(0)) += 1;
            }
            _ => {}
        }
    }

    println!("AC x {}", result.entry("AC").or_insert(0));
    println!("WA x {}", result.entry("WA").or_insert(0));
    println!("TLE x {}", result.entry("TLE").or_insert(0));
    println!("RE x {}", result.entry("RE").or_insert(0));
}
